apollo-router/src/executable.rs

```diff
@@ -677,6 +681,11 @@ impl Executable {
             }
         };
 
+        let threads = tokio::runtime::Handle::current().metrics().num_workers();
+        tracing::info!(threads, type="io", "thread pool");
```

Comment:
This is done here after we’ve initialized enough of the logging infrastructure. Unlike for compute threads there is no `source` here. The source could be:
* `$APOLLO_ROUTER_IO_THREADS`, if specified and `apollo_router::executable::main` is used
* Anything, if a custom binary defines its own `main` creating a Tokio runtime
* `$TOKIO_WORKER_THREADS` if specified and nothing set Tokio’s `worker_threads()`
* `available_parallelism()` is the eventual default